services/motion/builtin/builtin_test.go

```diff
@@ -763,7 +763,9 @@ func TestReplanning(t *testing.T) {
 
 		ctx, cancel := context.WithTimeout(ctx, 30.0*time.Second)
 		ma := newMoveAttempt(ctx, moveRequest)
-		ma.start()
+		err = ma.start()
+		test.That(t, err, test.ShouldBeNil)
```

Comment:
fix lint warning that we were not checking errors in tests.

Reply:
Do you have additional linters enabled? How were these passing the linter before?

Reply:
I do believe I somehow have additional linters enabled. 

I use neovim so maybe the vscode language server is catching these issues.